app/controllers/sufia/admin/admin_sets_controller.rb

```diff
@@ -52,6 +53,9 @@ def update
 
     def create
       if create_admin_set
+        @permission_template = permission_template
```

Comment:
For sure. I was concerned with conflicting with the method name, but I can use a local variable with a different name.